packages/broker/test/unit/configMigration.test.ts

```diff
@@ -130,35 +128,36 @@ const configWizardFull = {
     }
 }
 
-const pluginSchemas: Record<string,any> = {
-    websocket: WEBSOCKER_PLUGIN_CONFIG_SCHEMA,
-    mqtt: MQTT_PLUGIN_CONFIG_SCHEMA,
-    brubeckMiner: BRUBECK_MINER_PLUGIN_CONFIG_SCHEMA,
-}
-
-const validateTargetConfig = (config: any): void | never => {
+const validateTargetConfig = async (config: any): Promise<void> | never => {
     validateConfig(config, BROKER_CONFIG_SCHEMA)
     validateClientConfig(config.client)
-    Object.keys(config.plugins).forEach((name) => {
-        const pluginConfig = config.plugins[name]
-        const schema = pluginSchemas[name]
+    for (const pluginName of Object.keys(config.plugins)) {
+        const pluginConfig = config.plugins[pluginName]
+        const plugin = await createPlugin(pluginName, {
+            ...pluginConfig,
+            name: pluginName,
+            streamrClient: undefined,
+            apiAuthenticator: undefined,
+            brokerConfig: config
+        })
+        const schema = plugin.getConfigSchema()
         if (schema !== undefined) {
-            validateConfig(pluginConfig, schema, name)
+            validateConfig(pluginConfig, schema, pluginName)
         }
```

Comment:
This is already maybe checked by the `createPlugin` call above?

Reply:
Fixed